docs/pages/integration-test/masonry.js

```diff
@@ -79,3 +121,17 @@ export default function TestPage(): Node {
     </ColorSchemeProvider>
   );
 }
+
+export async function getServerSideProps(): Promise<{|
+  props: {| randomNumberSeeds: $ReadOnlyArray<number> |},
+|}> {
+  // This is used to ensure we're using the same dataset of realistic pins on the server and client
+  const randomNumberSeeds = Array.from({ length: REALISTIC_PINS_DATASET_SIZE }).map(() =>
```

Comment:
Is the seed to shuffle items on the client side?

Reply:
No, the opposite — by generating the array of random seeds as part of getServerSideProps, we ensure that we'll generate the same dataset of pin heights on both the server and the client.